zulipterminal/ui_tools/views.py

```diff
@@ -331,7 +331,7 @@ def __init__(self, topics_btn_list: List[Any], view: Any,
 
     @asynch
     def update_topics(self, search_box: Any, new_text: str) -> None:
-        if not self.view.controller.is_in_editor_mode():
+        if not self.view.controller.is_in_editor_mode() or not new_text:
```

Comment:
The existing code in `update_user_list` already handles empty search which is why this commit doesn't have any change for user search.